libs/framework/src/module.c

```diff
@@ -501,8 +494,6 @@ celix_status_t celix_module_loadLibraries(celix_module_t* module) {
             celixThreadMutex_lock(&module->handlesLock);
             module->bundleActivatorHandle = activatorHandle;
             celixThreadMutex_unlock(&module->handlesLock);
-        } else if (activatorHandle != NULL) {
-            celix_libloader_close(fwCtx, activatorHandle);
```

Comment:
This is indeed a double close. Suppose that we have two private libraries a.so and b.so and that a.so is an activator,
what will happen if a.so is opened successfully while loading b.so fails?

Reply:
Is it possible to create a test case for this, maybe using the a error injection when loading the b.so?

Reply:
Yes, I'll add a test case for this.